app-emulation/box64/autobuild/beyond

```diff
@@ -0,0 +1,4 @@
+abinfo "making libs executable..."
+chmod -v +x "$PKGDIR/usr/lib/box64-x86_64-linux-gnu/"*
+abinfo "moving default box64.conf to /usr/lib/binfmt.alternatives..."
+mv -v "$PKGDIR/etc/binfmt.d/box64.conf" "$PKGDIR/usr/lib/binfmt.alternatives/"
```

Comment:
This is bound to fail - you have not created the target directory yet.

Also, remove `"$PKGDIR"/etc/binfmt.d` after the move.